core/unit_test/TestAtomicOperations_complexdouble.hpp

```diff
@@ -45,6 +45,8 @@
 #include <TestAtomicOperations.hpp>
 
 namespace Test {
+// FIXME_SYCL atomic_fetch_oper for large types to be implemented
+#ifndef KOKKOS_ENABLE_SYCL
```

Comment:
Why do you remove it from the list if everything is being commented out?

Reply:
So that I have a better description for the FIXME.

Reply:
If you feel really strongly about it I can remove it but I prefer to have tests in `core/unit/CMakeLists.txt` that I have not looked at and found a better description.

Reply:
I prefer tests which completely not work to be removed in cmake

Reply:
You can have the comment in the CMakeLists.txt right next to the unit test you remove from the list

Reply:
OK. Changed.